packages/discord.js/typings/index.d.ts

```diff
@@ -4659,8 +4664,13 @@ export interface EscapeMarkdownOptions {
   underline?: boolean;
   strikethrough?: boolean;
   spoiler?: boolean;
-  inlineCodeContent?: boolean;
```

Comment:
I moved this down two lines so it can be in the same order as in `Util.js`. If wanted I can revert this.

Reply:
This is fine.